src/frontend/screens/MapScreen/index.tsx

```diff
@@ -114,6 +115,7 @@ export const MapScreen = () => {
 
         {isFinishedLoading && (
           <>
+            <RemoteDectionAlertsMapLayer />
```

Comment:
just checking since I may lack some context: this layer will be rendered _underneath_ the other layers - is that what's desired?

Reply:
I think that makes sense because (as it is now) the remote detection alerts are not clickable, while the other layers are.